src/olympia/reviewers/tests/test_views.py

```diff
@@ -2416,6 +2416,7 @@ def test_need_correct_reviewer_for_promoted_addon(self):
             'reject_multiple_versions',
```

Comment:
I was expecting an integration test here? You've tested all the units, but a reviewer clicks a button and a get/post? request comes to reviewers and what endpoint from reviewer tools is called with what arguments to forward to legal, creating or not creating a resolvable job?

Reply:
I added an integration test in the style we have in the same test class - that tests the request goes through, and then mocks the task call.